handout/a11y/key-concerns/keyboard-accessibility.md

```diff
@@ -0,0 +1,61 @@
+# Keyboard Accessibility
+
+Keyboard accessibility is the ability of your application to be interacted with using just a keyboard. The more streamlined the site can be used this way, the more keyboard accessible it is. Keyboard accessibility is one of the largest aspects of web accessibility since it targets:
+
+- those with motor disabilities who can't use a mouse
+- blind users who typically prefer to navigate websites with their keyboard
+- those who prefer not to use a mouse.
+
+
+## Focus
+
+Keyboard interaction is driven by something called *focus*. In web applications, only one element on a document has focus at a time, and keypresses are done using that element as the main point of context. This element can be accessed programmatically through the `document.activeElement` DOM method and visually, this is often represented by a glowing border around the element.
+
+![Focus border](./focus-border.png)
+
+
+### Tabbing
+
+The most common way of moving focus along the page is through the `tab` key. Elements will be traversed in the order they appear in the document outline - so that order must be carefully considered during development. By default, only links, buttons and form controls can receive keyboard focus.
+
+There may be cases where you'll want to change the default behaviour, this can be done through the `tabindex` attribute. The `tabindex` can be given the values:
+  - *less than zero* - to let readers know that an element should be focusable but not keyboard accessible
+  - *0* - to let readers know that that element should be accessible by keyboard
+  - *greater than zero* - to let readers know the order in which the focusable element should be reached using the keyboard. Order is calculated from lowest to highest.
+
```

Comment:
I think it's worth adding that one should use a focusable element wherever possible and tabindex as a last resort.
http://webaim.org/techniques/keyboard/tabindex